aws_adfs/login.py

```diff
@@ -31,6 +31,11 @@
     help='SSL certificate verification: Whether or not strict certificate\n'
          'verification is done, False should only be used for dev/test',
 )
+@click.option(
```

Comment:
Please, extend README.md by the new -aaadfs-ca-bundle option

Reply:
I have updated README.md with all new options on my branch, with the format as in the console.